db/schema.v0.sql

```diff
@@ -434,6 +434,19 @@ CREATE TABLE IF NOT EXISTS `application` (
   PRIMARY KEY (`id`)
 );
 
+-- -----------------------------------------------------
+-- Table `ical_key`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `ical_key` (
+  `key` CHAR(36) CHARACTER SET ascii NOT NULL,
+  `requester` CHAR(255) NOT NULL,
+  `name` CHAR(255) NOT NULL,
+  `type` ENUM('team', 'user') NOT NULL,
+  `time_created` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
MIght be a good idea to use UNIX_TIMESTAMP() instead since we already use unix time internally for Iris and Oncall.